pkg/azureclients/armclient/util.go

```diff
@@ -167,9 +168,16 @@ func DoHackRegionalRetryDecorator(c *Client) autorest.SendDecorator {
 					regionalErrStr = regionalError.Error()
 				}
 
-				klog.V(6).Infof("Send.sendRegionalRequest failed to get response from regional host, error: '%s'. Ignoring the result.", regionalErrStr)
+				klog.V(6).Infof("Send.sendRegionalRequest failed to get response from regional host, error: %q. Ignoring the result.", regionalErrStr)
 				return response, rerr
 			}
+			if regionalResponse.StatusCode == 200 && (regionalResponse.ContentLength == -1 || regionalResponse.ContentLength == 0) {
```

Comment:
it's not `response.StatusCode >= 200 && response.StatusCode < 300` ?

Reply:
let's replace `regionalResponse.StatusCode == 200` with `response.StatusCode >= 200 && response.StatusCode < 300` to keep align with above condition.

Reply:
Updated.